src/index.js

```diff
@@ -0,0 +1,213 @@
+// eslint-disable-next-line no-unused-vars
+import _, { indexOf, remove } from 'lodash';
```

Comment:
- [ ] Please, remove this unused import of _, indexOf and remove. There’s no reason to import modules you don’t use.

https://github.com/aliabbani/todo-list-review/blob/aba473f749722419d7be5cfb94a7922949046b5c/src/index.js#L2